body.tex

```diff
@@ -0,0 +1,440 @@
+\item all issues that are supposed to be included in it have been implemented and merged into master.
+\item a weekly build has been completed successfully.
+\item the product owner confirms that the weekly is good to go. \footnote{This is not a full test, but just ensures that there are no regressions nor problems at a first glance.}
+\end{itemize}
+
+In case a weekly is presenting problems that may affect the release, the DMCCB will decide if delay the release start or kick--off the process and fix the problems in forthcoming release candidates.
+
+
+Following steps have to be used to create the first release candidate:
+
+\begin{itemize}
+\item create a release candidate in GitHub (Codekit\footnote{Code management toolkit \url{https://github.com/lsst-sqre/sqre-codekit}})
+\item execute a build (using Jenkins job run-rebuild)
+\item create and publish the packages. For the Science Pipelines, EUPS packages are created and published in the package repository:
+   \begin{itemize}
+   \item source packages (using run-publish Jenkins job)
+   \item binaries packages (using tarball Jenkins job)
+   \end{itemize}
+\item create the distribution package. Still for the Science Pipelines, a docker image is generate and made available in the docker repository:
+   \begin{itemize}
+   \item distribution generated and published using build-stack Jenkins job
+   \end{itemize}
+\end{itemize}
+
+
+This procedure should be applied for all software products in the DM product tree.
+
+The parameters required are:
+\begin{itemize}
+\item GIT\_REFs: the existing Git references, for example "w.2018.52". This is the stable weekly build identified.
+\item GIT\_TAG: the release candidate to be generated, for example "M.n.p.rc1"
+\end{itemize}
+
+For the Science Pipelines, the above steps are collected in one single Jenkins job:
+
+\begin{verbatim}
+releases/official-release
+\end{verbatim}
+
+Note that for all software product using EUPS, like for example the Science Pipelines, release candidates need to start with a $v$.
+
+\subsection{Other Related Artifacts}
+
+Together with the creation of the release candidate, other artifacts may need to be branched, and possibly have a first release candidate also:
+
+\begin{itemize}
+\item Documentation: to consolidate all information relevant for the new release. This includes the release note.
+As an example, for Science Pipelines, the package \textit{pipelines\_lsst\_io} need to be branched.
+\item  Environment definition packages, like for example packages containing conda environment files (YAML).
+\item  Distribution packages; still referring to the Science Pipelines, the package \textbf{lsst}  where \textit{newinstal.sh} is developed, need to be branched also.
+\item Test data packages.
+\item Other packages not tagged automatically, but required to be inline with the released software.
+\end{itemize}
+
+
+\subsection{Release Candidates Validation} \label{sec:rcvalidation}
+
+The release candidate needs to be validated.
```

Comment:
Who is responsible for this activity?

Reply:
I try to add some text to clarify.
I think that T/CAM and product owner shall be responsible for this, but I am open to other suggestions.
For sure, the release manager can validate the release, apart from ensuring that the release process has been followed.

Reply:
Release manager can, or can't?

Reply:
can't